js/thumbnail.js

```diff
@@ -0,0 +1,20 @@
+const picturesList = document.querySelector('.pictures');
+const pictureTemplate = document.querySelector('#picture').content.querySelector('.picture');
```

Comment:
Я бы попросил все переменные которые содержат элементы - называть с **Element** на конце
```suggestion
const picturesElement = document.querySelector('.pictures');
const pictureTemplateElement = document.querySelector('#picture').content.querySelector('.picture');
```

Reply:
Исправил